_includes/templates/install/ubuntu-queue-pub-sub.md

```diff
@@ -0,0 +1,42 @@
+#### Google Pub/Sub Configuration
+
+To access Pub/Sub service, you first need to create an [Google cloud account](https://cloud.google.com/).
+
+To work with Pub/Sub service you will need to create a project using [this instruction](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
```

Comment:
I'm not sure that this Guide includes the steps for enabling apis https://cloud.google.com/apis/docs/getting-started#enabling_apis that required for the pubsub queue setup.

Without enabling the pub/sub APIs I received the next error:

		at org.thingsboard.server.ThingsboardServerApplication.main(ThingsboardServerApplication.java:38)
Caused by: io.grpc.StatusRuntimeException: PERMISSION_DENIED: Cloud Pub/Sub API has not been used in project ######## before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/pubsub.googleapis.com/overview?project=741534699124 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
	at io.grpc.Status.asRuntimeException(Status.java:533)
	... 24 common frames omitted